### PR TITLE
windows platform fixes [13947]

### PR DESCRIPTION
**Pull request by MiguelBarro**
+ swig files must ignore some non-exported APIs (unix exports everything by policy)
+ fix examples CMake framework to handle dlls exports
+ hinting workaround for python [importing issues](https://bugs.python.org/issue46276)

### CODE REVIEW

File: .github/workflows/test.meta [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "names":
+    {
+        "fastrtps" :
+        {
+            "cmake-args": [
+                "-DTHIRDPARTY=ON",
+                "-DSECURITY=ON",
```
Comment (MiguelCompany) [inline .github/workflows/test.meta:8]: Why adding security? I would do that on a follow-up if need be.

File: fastdds_python/src/swig/fastdds/dds/core/condition/StatusCondition.i [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+%{
+#include "fastdds/dds/core/condition/StatusCondition.hpp"
+%}
+
+// Ignore StatusCondition constructor
+%ignore eprosima::fastdds::dds::StatusCondition::StatusCondition;
```
Comment (MiguelCompany) [inline fastdds_python/src/swig/fastdds/dds/core/condition/StatusCondition.i:20]: Remove this file and move this to `Condition.i`

File: fastdds_python/src/swig/fastdds/dds/subscriber/DataReader.i [hunk 3]
```diff
@@ -32,4 +32,6 @@
     }
 }
 
+%ignore eprosima::fastdds::dds::DataReader::type;
+
```
Comment (MiguelCompany) [inline fastdds_python/src/swig/fastdds/dds/subscriber/DataReader.i:36]: Remove this. The method is exported by Fast DDS now.

File: fastdds_python/src/swig/fastdds/dds/topic/Topic.i [hunk 4]
```diff
@@ -16,4 +16,9 @@
 #include "fastdds/dds/topic/Topic.hpp"
 %}
 
+// Ignore get_inconsistent_topic_status which is not exported
+%ignore eprosima::fastdds::dds::Topic::get_inconsistent_topic_status;
```
Comment (MiguelCompany) [inline fastdds_python/src/swig/fastdds/dds/topic/Topic.i:20]: I would add the export to Fast DDS and remove this line.

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Fix DCO and see my comments below

File: fastdds_python_examples/HelloWorldExample/HelloWorldExample.py [hunk 5]
```diff
@@ -14,20 +14,22 @@
...
 import argparse
-import signal
 from threading import Condition
 
+# until https://bugs.python.org/issue46276 is not fixed we can apply this
+# workaround on windows
+#if os.name == 'nt':
+#    import win32api
+#    win32api.LoadLibrary('HelloWorld')
+
```
Comment (MiguelCompany) [inline fastdds_python_examples/HelloWorldExample/HelloWorldExample.py:26]: Should we uncomment these lines?

**Review by MiguelCompany — APPROVED** [review]
Just a question, but LGTM

File: fastdds_python_examples/HelloWorldExample/HelloWorldExample.py (hunk 5 above)
Comment (richiware) [reply fastdds_python_examples/HelloWorldExample/HelloWorldExample.py]: Yes and uploaded.

**Review by MiguelCompany — DISMISSED** [review]
LGTM

**Review by EduPonz — APPROVED** [review]

**Pull request merged**